README.md

```diff
@@ -72,6 +75,7 @@ fastify.register(require('fastify-rate-limit'), {
 
 - `global` : indicates if the plugin should apply the rate limit setting to all routes within the encapsulation scope
 - `max`: is the maximum number of requests a single client can perform inside a timeWindow. It can be a sync function with the signature `(req, key) => {}` where `req` is the Fastify request object and `key` is the value generated by the `keyGenerator`. The function **must** return a number.
+- `ban`: is the maximum number of 429 responses to return to a single client before returning 403. When the ban limit is exceeded the context field will have `ban=true` in the errorResponseBuilder
```

Comment:
I mean that the ban parameter works in a fuzzy way in a distributed environment, like using the localstore.
(The text I posted is copy-pasted from the `redis` param)

Sorry for the misunderstanding

```suggestion
- `ban`: is the maximum number of 429 responses to return to a single client before returning 403. When the ban limit is exceeded the context field will have `ban=true` in the errorResponseBuilder. This parameter is an in-memory counter and could not works properly in a distributed environment.
```

Reply:
Updated accordingly, agree it is a bit fuzzy. Though it is calculated based on a `current` and `max` while in the time window, so if they use LocalStore then it can be meaningless, if they use Redis then at least `current` is stored in a central fashion so the parameter should have meaning then.